_data/meltano/extractors/tap-google-analytics/meltanolabs.yml

```diff
@@ -85,28 +95,28 @@ settings:
     2. Next, visit the [Google Analytics API](https://console.developers.google.com/apis/api/analytics.googleapis.com/overview) dashboard, make sure that the project you used in the previous step is selected, and enable this API as well.
 
       ![Screenshot of Google Analytics API](/assets/images/tap-google-analytics/05-ga-api.png)
-  kind: file
```

Comment:
`file` is a valid kind https://github.com/meltano/meltano/blob/658444f1a839911a218dedfaec26844bc8262ac4/src/meltano/schemas/meltano.schema.json#L794